modules/Bio/EnsEMBL/Production/Pipeline/Webdatafile/WebdataFileBootstrap.pm

```diff
@@ -0,0 +1,214 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+ Bio::EnsEMBL::Production::Pipeline::Webdatafile::WebdataFileBootstrap;
+
+=head1 DESCRIPTION
+  Compute bootstrap step for webdatafile dumps
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::Webdatafile::WebdataFileBootstrap;
```

Comment:
Package name should be renamed to something else, related to comments from Code Review today.  Thanks!

Reply:
suggesting WebdataFileGenomeAssemblyInfo ?

Reply:
Fine, but that makes for a long name; the "WebdataFile" bit at the start is redundant because that in the path already, so maybe drop that?

Reply:
sure will do the same for other modules to